src/test/java/igrad/logic/commands/ClearCommandTest.java

```diff
@@ -0,0 +1,32 @@
+package igrad.logic.commands;
+
+import static igrad.logic.commands.CommandTestUtil.assertCommandSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import igrad.model.CourseBook;
+import igrad.model.Model;
+import igrad.model.ModelManager;
+import igrad.model.UserPrefs;
+import igrad.testutil.TypicalPersons;
+
+public class ClearCommandTest {
+
+    @Test
+    public void execute_emptyCourseBook_success() {
+        Model model = new ModelManager();
+        Model expectedModel = new ModelManager();
+
+        assertCommandSuccess(new ClearCommand(), model, ClearCommand.MESSAGE_SUCCESS, expectedModel);
+    }
+
+    @Test
+    public void execute_nonEmptyCourseBook_success() {
+        Model model = new ModelManager(TypicalPersons.getTypicalCourseBook(), new UserPrefs());
```

Comment:
you wouldn't have to do this, if you used static imports, as per the original code;
`Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());`

(please refer to comment above)